src/gui/SquareSvgWidget.cpp

```diff
@@ -26,3 +33,21 @@ int SquareSvgWidget::heightForWidth(int width) const
 {
     return width;
 }
+
+// The overridden logic allows to keep the SVG image as square and centered by width and height.
+void SquareSvgWidget::resizeEvent(QResizeEvent*)
+{
+    Q_ASSERT(parentWidget());
+    auto containerRect = parentWidget()->contentsRect();
```

Comment:
Do a null check here, there is no guarantee the widget will have a parent and you will instacrash here if it doesn't. Assert only gets hit in debug build

Reply:
Sure, add the checking for null.